coherent_rates/system.py

```diff
@@ -345,3 +371,35 @@ def get_cl_operator(
         "basis": TupleBasis(converted_potential["basis"], converted_potential["basis"]),
         "data": matrix.ravel(),
     }
+
+
+_BT0 = TypeVar("_BT0", bound=BasisWithTimeLike[Any, Any])
+
+
+def get_gaussian_fit(
+    values: ValueList[_BT0],
+) -> tuple[float, float, float, float]:
```

Comment:
return a dataclass (see PeriodicSystem for an example). This prevents you getting the order messed up